website/static/js/pages/view-file-page.js

```diff
@@ -11,3 +14,33 @@ new FileRevisions(
     window.contextVars.file,
     window.contextVars.currentUser.canEdit
 );
+
+$(document).ready(function() {
+    // Treebeard Files view
+    $.ajax({
+        url: nodeApiUrl + 'files/grid/'
+    })
+        .done(function (data) {
+            new fileBrowser(data);
+        });
+
+    var panelToggle = $('.panel-toggle');
```

Comment:
This seems to be an exact copy of the dataverse view file page code. Can this be DRY'd up?